jina/resources/health_check/pod.py

```diff
@@ -13,7 +13,7 @@ def check_health_pod(addr: str, timeout: int = 600):
 
     channel = grpc.insecure_channel(addr)
     stub = jina_pb2_grpc.JinaControlRequestRPCStub(channel)
-    request = ControlRequest(command='STATUS')
+    request = ControlRequest(command="STATUS")
```

Comment:
I believe there is some already built function in `GRPCConnectionPool`. We should use the same `Pods` actually do to check readiness

Reply:
```python
GrpcConnectionPool.send_request_sync(
      request=ControlRequest('STATUS'), target=host, https=True
)
```

Reply:
Ah okay got it, I actually inspire myself from this function